daemon/graphdriver/windows/windows.go

```diff
@@ -47,6 +48,10 @@ type Driver struct {
 	active map[string]int
 }
 
+func ensureDriverInterface(d *Driver) graphdriver.DiffGetterDriver {
```

Comment:
the convention to do that is to declare an unamed variable:

```go
var _ graphdriver.DiffGetterDriver = &Driver{}
```

We have other declarations like that in packages like the logger, it would be more consistent if we did the same here.